kustomize/base/wf_crd/wf_crd.yaml

```diff
@@ -7,7 +7,23 @@ spec:
   # group name to use for REST API: /apis/<group>/<version>
   group: flyte.lyft.com
   # version name to use for REST API: /apis/<group>/<version>
-  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                cronSpec:
+                  type: string
+                image:
+                  type: string
+                replicas:
+                  type: integer
```

Comment:
I don't think we have these properties on the object... this can be left empty. right?

Reply:
We are not quite familiar with this part. However, if we don't include anything in the schema, it still broke. That's why we included an empty schema. If you think this can be left empty, please go ahead.

Reply:
this worked for me:
```
  versions:
  - name: v1alpha1
    schema:
      openAPIV3Schema:
        description: 'FlyteWorkflow: represents one Execution Workflow object'
        type: object
    served: true
    storage: true
```